stop.bash

```diff
@@ -19,10 +19,13 @@ if [ "$1" = "release" ]; then
     EXECMODE="release"
     shift
 fi
-if [ "$1" = "CD" ]; then
+if [ "$1" = "release" ] || [ "$1" = "CD" ]; then
     EXECMODE="release"
     shift
 fi
+if [ "$EXECMODE" == "CD" ]; then
```

Comment:
== won't work in bash, also it is a duplicate of the test in line 22

Reply:
== does work in bash, but for consistency we should stick to either one and therefore I ll change it to a single =.

It is not a duplicate of line 22 as it evaluates, if CD is set as default mode in the settings.bash. The difference in content to the if clause in line 22 is just the shift command, which we could agree on removing for this script as it is not needed, but then I'd remove it for all exec modes.